pyproject.toml

```diff
@@ -47,7 +47,7 @@ boto3 = ">=1.35.43"  # AWS S3 for Data Layer S3 plugin
 chiabip158 = ">=1.5.2"  # bip158-style wallet filters
 chiapos = ">=2.0.10"  # proof of space
 chia-puzzles-py = ">=0.20.1"
-chia_rs = ">=0.21.1"
```

Comment:
I don't think we support `chia_rs-0.21.0`. is it not possible to constrain the 3rd version number?